trestle/common/const.py

```diff
@@ -559,6 +559,8 @@
 
 CONTROL_IMPLEMENTATION = 'control-implementation'
 
+AGGREGATES = 'aggregates'
+
 IMPLEMENTED_REQUIREMENT = 'implemented-requirement'
```

Comment:
There was another parameter property in FedRAMP profile for "alternative". Is that not being handled? For alternative prop also we should just show the alternative parameter name (like aggregates) and. not show the value as the value comes from another (main) parameter.

Reply:
Done